packages/ui/src/composites/wui-account-button/index.ts

```diff
@@ -49,7 +49,8 @@ export class WuiAccountButton extends LitElement {
             address=${this.address}
           ></wui-avatar>
           <wui-text variant="paragraph-600" color="inherit">
-            ${UiHelperUtil.getTruncateString({
+            ${this.address &&
```

Comment:
Let's not use the `&&` (AND) operator and instead use the `?` (ternary) operator. The AND operator might render the left-hand side values to the DOM, when it's `truty` but looks like "falsy". 
See: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Even though we could be sure about the type definition of the `this.address`, the developer shouldn't be responsible for checking if it's not causing this issue or not.